### PR TITLE
Adding new exceptions

### PR DESCRIPTION
**Pull request by ductiletoaster**
Adding a bunch of exceptions and making the structure and tests simpler

### CODE REVIEW

File: packages/pixeloven-exceptions/src/Node/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default as NodeProcessException } from "./ProcessException";
```
Comment (jimmynotjim) [inline packages/pixeloven-exceptions/src/Node/index.ts]: Did you mean to update this exported name to match the new class name in the file above??

Comment (jimmynotjim) [reply packages/pixeloven-exceptions/src/Node/index.ts]: NVM, I see what you're doing with the dir structure now

Comment (ductiletoaster) [reply packages/pixeloven-exceptions/src/Node/index.ts]: Eventually I want to do Node.ProcessException but that would require me to make other changes that I wasn't willing to make yet

Comment (jimmynotjim) [reply packages/pixeloven-exceptions/src/Node/index.ts]: I had a feeling that's where you're headed looking at all the dirs as a collection.

**Pull request merged**